Auth0.xcodeproj/project.pbxproj

```diff
@@ -2639,8 +2654,12 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Carthage/Build/iOS",
 				);
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"$(inherited)",
+					"WEB_AUTH_PLATFORM=1",
+				);
 				INFOPLIST_FILE = Auth0Tests/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
Since SPM defines the same platform versions for all targets, the test targets were changed here to match the same minimum platform versions as the library. Previously, the test targets had higher requirements to avoid having to guard the code everywhere a newer API was used (generally crypto APIs). So the code guards were added as well.